src/reducers/sources.js

```diff
@@ -92,7 +92,9 @@ function update(
         url: action.source.url
       };
 
-      prefs.pendingSelectedLocation = location;
+      if (action.source.url) {
+        prefs.pendingSelectedLocation = location;
+      }
```

Comment:
@Doomwyte This change allows the editor to select the first available tab upon refresh; `prefs.pendingSelectedLocation` saves the last source (and its location) to localStorage so that it can be selected next time around.  If the source doesn't have a URL, however, we don't want that to be saved.